WordPress/Classes/ViewRelated/Blog/Blog Details/BlogDetailsViewController.h

```diff
@@ -74,6 +74,8 @@ typedef NS_ENUM(NSInteger, QuickStartTourElement) {
     QuickStartTourElementNotifications = 23,
     QuickStartTourElementSetupQuickStart = 24,
     QuickStartTourElementRemoveQuickStart = 25,
+    QuickStartTourElementMediaScreen = 26,
+    QuickStartTourElementMediaAdd = 27,
```

Comment:
MediaUpload might be a better name, wdyt?
```suggestion
    QuickStartTourElementMediaUpload = 27,
```

Reply:
Done! 724a60ded182b0b18ee6a76b41d839b6ae7a77b2